cpprevolve/revolve/brains/CMakeLists.txt

```diff
@@ -18,19 +18,23 @@ find_package(Boost REQUIRED COMPONENTS system)
 # Find Eigen3 - A lightweight C++ template library for vector and matrix math
 find_package(Eigen3 REQUIRED)
 
+find_package(MultiNEAT REQUIRED)
+
 # Find NLOpt - Non Linear Optimization
 pkg_check_modules(NLOpt REQUIRED nlopt>=2.4)
 
 # Find Limbo - LIbrary for Model-Based Optimization
 set(LIMBO_DIR ${CMAKE_SOURCE_DIR}/thirdparty/limbo)
+set(MULTINEAT_DIR ${CMAKE_SOURCE_DIR}/thirdparty/MultiNEAT)
 set(LIMBO_DEFINES USE_NLOPT)
 
 add_library(revolve-controllers SHARED ${CONTROLLER_SRCS})
 add_library(revolve-learners SHARED ${LEARNER_SRCS})
 
 target_include_directories(revolve-controllers
         PUBLIC ${EIGEN3_INCLUDE_DIR}
-        PUBLIC ${Boost_INCLUDE_DIRS})
+        PUBLIC ${Boost_INCLUDE_DIRS}
+        PUBLIC ${MULTINEAT_DIR}/src)
```

Comment:
No

Reply:
Instead there should be a `target_link_library(revolver-controllers MultiNEAT::MultiNEAT)`